DashWallet/Sources/UI/Explore Dash/Views/DWExploreTestnetContentsView.m

```diff
@@ -245,4 +256,62 @@ - (NSString *)subtitle {
 - (void)setSubtitle:(NSString *)subtitle {
     self.descLabel.text = subtitle;
 }
+
+- (void)addContent:(UIView *)view {
+    UIView *last = [[_contentStack arrangedSubviews] lastObject];
+    [_contentStack setCustomSpacing:10 afterView:last];
+    [_contentStack addArrangedSubview:view];
+    _contentStack.layoutMargins = UIEdgeInsetsMake(0, 0, 10, 0);
+    _contentStack.layoutMarginsRelativeArrangement = YES;
```

Comment:
I'm not sure if there isn't a better way to add some space around the APY element. Perhaps we can just put it inside another view and handle insets in there?